tests/basic.t

```diff
@@ -65,49 +65,94 @@ if [ $? -eq 1 ]; then
   TEST glusterd
 fi
 
-# Create gluster volume
+##### Create Gluster Volume Test Start #####
 TEST gluster vol create ${VOLNAME} ${HOST}:${BRKDIR} force
 
 # Start the volume
 TEST gluster vol start ${VOLNAME}
+##### End #####
 
 
-# Test gfapi access
+##### Gfapi Access Test Start #####
 TEST ./tests/gfapi-test ${VOLNAME} ${HOST}
+##### End #####
 
-# Start gluster-blockd.service
+
+##### Restart gluster-blockd.service Test Start #####
 systemctl daemon-reload
-TEST systemctl restart gluster-blockd.service
+TEST systemctl restart gluster-block-target.service gluster-blockd.service
 sleep 1;
```

Comment:
@lxbsz , lets make this sleep time to 5 seconds.

I see:
TEST : systemctl restart gluster-block-target.service gluster-blockd.service
line 83 : OK                                                           
                                                                      
TEST : gluster-block create hosting-volume/block-volume 192.168.124.147 1MiB                                          
client create failed: RPC: Remote system error - Connection refused, unix addr /var/run/gluster-blockd.socket         
line 90 : NOT OK

Lets give daemon some more time to finish the init stuff

Reply:
Is 3 seconds okay? or 5 ?

Reply:
Its okay with 5 seconds wait. These are just tests, lets give sometime for it.